Tests/SonarQube.TeamBuild.PreProcessor.Tests/TargetsInstallerTests.cs

```diff
@@ -128,11 +363,17 @@ private static void CreateDummySourceTargetsFile(string sourceTargetsContent1)
 
         private void InstallTargetsFileAndAssert(string expectedContent, bool expectCopy)
         {
-            var installer = new TargetsInstaller();
-            var logger = new TestLogger();
-            installer.InstallLoaderTargets(logger, WorkingDirectory);
+            var localLogger = new TestLogger();
```

Comment:
This method is called multiple times per test and the shared `logger` contained more messages than expected